2006Scape Server/src/main/java/com/rs2/game/content/combat/CombatAssistant.java

```diff
@@ -30,6 +30,8 @@
 import com.rs2.world.Boundary;
 import com.rs2.world.clip.PathFinder;
 
+import static com.rs2.game.content.StaticItemList.GRANITE_MAUL;
```

Comment:
We should probably just import StaticItemList because we'll need it for other magic numbers in this file

Reply:
Swarm is now ambiguous btw, since there's a swarm item and swarm NPC, so swarm will need to be updated to mention StaticNpcList most likely.